include/nbl/asset/IDescriptorSetLayout.h

```diff
@@ -196,6 +304,7 @@ class NBL_API IDescriptorSetLayout : public virtual core::IReferenceCounted
 				}
 			}
 
+			std::fill_n(m_descriptorCount, EDT_COUNT, 0ull);
```

Comment:
this can be retrieved from `SBindingRedirects` is the offsets will be inclusive scanned

Reply:
Resolved.